Holovibes/includes/gui/windows/panels/panel.hh

```diff
@@ -70,6 +75,7 @@ class Panel : public QGroupBox
   protected:
     MainWindow* parent_;
     Ui::MainWindow* ui_;
+    api::Api& api_;
```

Comment:
Why do panels need an api reference ? Isn't the api a singleton available everywhere ?
If the issue is typing api::Api::instance() everytime, shouldn't this be fixed by a macro ?